api/DOMTokenList.json

```diff
@@ -220,7 +220,7 @@
               "version_added": "32"
             },
             "safari": {
-              "version_added": "10"
+              "version_added": "10.1"
```

Comment:
`iterable<DOMString>` is from https://trac.webkit.org/changeset/203728/webkit, WebKit 603.1.1. That confirms this change. But safari_ios should then be "10.3". I'll suggest edits.